sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -29,6 +27,8 @@ import scala.concurrent.{Await, Promise}
 import scala.sys.process.{Process, ProcessLogger}
 import scala.util.{Random, Try}
 
+import com.google.common.base.Charsets
```

Comment:
I mean if importing `com.google.common.base.Charsets.UTF_8` here, you can use `UTF_8` directly in the codes. And when we decide to switch to `java.nio.charset.StandardCharsets.UTF_8`, we only need to update the import.